tsconfig.base.esm.json

```diff
@@ -2,6 +2,7 @@
   "extends": "./tsconfig.es5.json",
   "compilerOptions": {
     "module": "ES6",
-    "moduleResolution": "node"
+    "moduleResolution": "node",
+    "target": "ES6"
```

Comment:
This should have been defined in `tsconfig.es5.json`. Would you mind removing this unnecessary change?